functions/src/send-email-to-manager.ts

```diff
@@ -68,13 +68,18 @@ export const sendEmailToManager = functions.firestore
      */
     let emailSent = false;
     await firebase.firestore().runTransaction(async transaction => {
+      //Make sure we didn't send a mail already
+      if (vote.emailToManagerSent) {
+        console.info("Email already sent to manager, aborting");
+        return;
+      }
       if (!emailSent) {
         await mailgunClient.messages().send(message);
         emailSent = true;
       }
       transaction.update(voteSnapshot.ref, {
         emailToManagerSent: true,
-        voter: "*REDACTED*"
+        fullName: "*REDACTED*"
```

Comment:
Isn't the email there too?

Reply:
I just checked and the answer is yes. We need to be careful about this. I think we need to separate personal information from non-personal information in order to never miss field when anonymizing.